gobblin-data-management/src/main/java/gobblin/data/management/copy/publisher/CopyDataPublisher.java

```diff
@@ -61,6 +62,9 @@ public CopyDataPublisher(State state) throws IOException {
     String uri = this.state.getProp(ConfigurationKeys.WRITER_FILE_SYSTEM_URI, ConfigurationKeys.LOCAL_FS_URI);
 
     this.fs = FileSystem.get(URI.create(uri), conf);
+
+    FileAwareInputStreamDataWriterBuilder.setJobSpecificOutputPaths(state);
```

Comment:
I feel these are writer properties which the source really should not care about.